### DISCUSSION TITLE
Localized History display not handled by Sparkle

### DISCUSSION THREAD

**Discussion by bonnaphil** [sparkle-project/Sparkle#2144] [Q&A]
## Summary

Localized History display not handled by Sparkle

## Version

Latest Sparkle released version

## Bug description

Hi,

I am the French Localizer of GraphicConverter by LemkeSoftware. The latest version of Sparkle is embedded in the latest release of GraphicConverter 11.6.1. The problem is when a click is done one the Version history button (Historique des versions) below :

![History button](https://user-images.githubusercontent.com/24955489/172605467-ec164e29-ad01-4635-8ac7-1cb4fec2e1a5.jpg)

The displayed History page is always the English history page and not the French history page that is also available. So maybe a link problem in the current Sparkle version.

**Comment by zorgiepoo** [comment]
```xml
<item>
    <title>Version 11.6.2 Build 5524 BETA</title>
				<sparkle:releaseNotesLink xml:lang="en">https://www.lemkesoft.info/sparkle/graphicconverter/notes/beta.html</sparkle:releaseNotesLink>
	<sparkle:version>5524</sparkle:version>
	<sparkle:minimumSystemVersion>10.13.0</sparkle:minimumSystemVersion>
	<sparkle:shortVersionString>11.6.2</sparkle:shortVersionString>
	<sparkle:channel>beta</sparkle:channel>
    <pubDate>7 June 2022 8:51:00 +0000</pubDate>
    <enclosure url="https://www.lemkesoft.org/beta/gc11_build5524.zip" sparkle:edSignature="FCE6Xtq4T086l5PbccAllCTjg2/B0jkqKKDrM54SkZjKbCcpBnewwRKF1qbXYPJqPVCJt62CKKNQY9n2j3FfBQ==" sparkle:dsaSignature="MC4CFQCKU8pCECbv4fU12nhWuHMmPc0ccQIVANiItqRoPmsukdU5NWY6lfyK1TfR" length="300188520" type="application/octet-stream" />
</item>
```

Your latest (beta) appcast item is missing `<sparkle:releaseNotesLink xml:lang="fr"></sparkle:releaseNotesLink>` which Sparkle would default to if the user's system language is set to French. Your alert dialog is showing you're on a build of 11.6.2 I believe..

You can also consider setting `<sparkle:fullReleaseNotesLink>` to [your full release notes](https://www.lemkesoft.de/en/products/graphicconverter/overview-of-new-features-and-versions) instead; I don't know if you localize that webpage though (but the `<sparkle:fullReleaseNotesLink>` element should be `xml:lang` localizable as well). See [the documentation](https://sparkle-project.org/documentation/publishing/#full-release-notes).
